static/app/components/charts/optionCheckboxSelector.tsx

```diff
@@ -0,0 +1,186 @@
+import {Component, createRef, Fragment} from 'react';
+import styled from '@emotion/styled';
+import isEqual from 'lodash/isEqual';
+
+import {InlineContainer, SectionHeading} from 'app/components/charts/styles';
+import CheckboxFancy from 'app/components/checkboxFancy/checkboxFancy';
+import DropdownBubble from 'app/components/dropdownBubble';
+import DropdownButton from 'app/components/dropdownButton';
+import {DropdownItem} from 'app/components/dropdownControl';
+import DropdownMenu from 'app/components/dropdownMenu';
+import Tooltip from 'app/components/tooltip';
+import Truncate from 'app/components/truncate';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {SelectValue} from 'app/types';
+
+const defaultProps = {
+  menuWidth: 'auto',
```

Comment:
should we limit this so the menu doesn't become super wide if the column name is really wide?

Reply:
Yup, we're actually limiting the option labels using the `StyledTruncate` component right now in `optionCheckboxSelector`! The current max is set at 60 characters